client/packages/system/src/Report/components/ReportArgumentsModal.tsx

```diff
@@ -68,17 +69,25 @@ export const ReportArgumentsModal: FC<ReportArgumentsModalProps> = ({
         />
       }
     >
-      <JsonForm
-        data={data}
-        jsonSchema={report.argumentSchema.jsonSchema}
-        uiSchema={report.argumentSchema.uiSchema}
-        isError={false}
-        isLoading={false}
-        setError={err => setError(err)}
-        updateData={(newData: JsonData) => {
-          setData(newData);
-        }}
-      />
+      <>
+        <Typography
```

Comment:
Thought it would be fine to show this on all the arg modals since it is for filtering